README.md

```diff
@@ -89,6 +89,7 @@ for all of its commands and options internally, but some specific concepts
 have additional documents describing them:
  * [Sessions](docs/cli-sessions.md)
  * [Extended headers](docs/cli-xheaders.md)
+ * [Exit codes](docs/cli-return-codes.md)
```

Comment:
so maybe `docs/cli-exit-codes.md`? as i see it is always "exit" in that PR except file naming